dietpi/patch_file

```diff
@@ -16,6 +16,9 @@
 	# - /boot/dietpi/patch_file [$G_DIETPI_VERSION_SUB]
 	#////////////////////////////////////
 
+	# Allow dietpi user to run sudo commands as any user
+	echo 'dietpi ALL=(ALL) NOPASSWD: ALL' > /etc/sudoers.d/dietpi
```

Comment:
Currently: https://github.com/MichaIng/DietPi/blob/master/dietpi/func/dietpi-set_software#L407
- `dietpi ALL=NOPASSWD: ALL`

This does not allow becoming any user? I'll test later. Else we should change that in the underlying script as well.

Reply:
No, it doesn't.  From the `sudoers` man page:
```
If no Runas_Spec is specified the command may be run as root and no group may be specified.
```

Reply:
Ah okay. Then you last commit makes sense. As long as `sudo sudo -u ...` can be used to become any user, this does not really add any permissions, it just allows to become any user without this awkward `sudo` concatenation 😄.